src/content/blog/2024/core_retreat_Feb_2024.mdx

```diff
@@ -0,0 +1,110 @@
+import viking5 from '@assets/images/blog/core_retreat_Feb_2024/viking5.jpg';
+import viking6 from '@assets/images/blog/core_retreat_Feb_2024/viking6.jpg';
+import viking7 from '@assets/images/blog/core_retreat_Feb_2024/viking7.jpg';
+import viking8 from '@assets/images/blog/core_retreat_Feb_2024/viking8.jpg';
+import fin from '@assets/images/blog/core_retreat_Feb_2024/fin.jpg';
+
+
+import { Image } from 'astro:assets';
+
+<Image src={work_pic1} alt="core team working at retreat" />
+
+Between the 5th and 8th of February, the core team and members of the Steering group met in icy Stockholm to discuss upcoming challenges and improvements to the community. The topics and desicions are outlined below.
+
+
+# Structure and Governance
+
+-   **Funding**
+
+    As many open-source communities, funding is an important topic. The nf-core community has applied for another grant from the Chan Zuckerberg Initiative, but we will not hear about a decision until later in the year. Until then, the community is run on volunteer work by all of you and supported by different Universities and institutions.
+
+-   **nf-core trademark**
+
+    Currently, the nf-core logo is not trademarked. During the retreat the core team and steering group discussed the benefits and drawbacks of trademarking the nf-core logo.
+
+-   **nf-core working groups**
+
+    An exiting development is that nf-core wants to establish working groups / SIGs / IGs / COSIs to encourage people within the nf-core community with similar interest to meet and work together. More information on this new initiative will be presented in the bytesize talk on February 20th [link to the bytesize page].
+
+# Outreach and Community
+
+-   **Training and Documentation overhaul**
+
+    One of the main goals of nf-core is to provide easy to use and well documented tools and pipelines. While the pipeline documentations are in a good state, the docs for tools and training are a bit scattered throughout the website and at times not very accessible. You can look forward to a complete overhaul of the docs page towards a more modular setup, which will also help putting together more targeted training for different levels of knowledge. Be aware that this is a major project and will likely require a good amount of time.
+
+-   **Publication of a nf-core v2.0 paper**
+
+    It has been 4 years since the first nf-core paper was published [insert gif or something] and the community has changed masively since then. Not only has it tremendously grown in size, but we also experienced a lot of technical changes including the update to Nextflow DSL2, the introduction of modules and subworkflows and upgrades to nf-core/tools. To reflect these changes, the core team would like to publish an updated state of nf-core paper. A first draft is in the works and will also emphasize the work done in the community build around FAANG that sparked the idea about creating work groups.
```

Comment:
```suggestion
    It has been 4 years since the first nf-core paper was published [insert gif or something] and the community has changed massively since then. Not only has it tremendously grown in size, but we also experienced a lot of technical changes including the update to Nextflow DSL2, the introduction of modules and subworkflows and upgrades to nf-core/tools. To reflect these changes, the core team would like to publish an updated state of nf-core paper. A first draft is in the works and will also emphasize the work done in the community build around FAANG that sparked the idea about creating work groups.
```